### DISCUSSION TITLE
Is there a setup that allows editing the (English) base language of Windows RC files?

### DISCUSSION THREAD

**Discussion by JustSwiftIt** [WeblateOrg/weblate#13095] [Q&A]
I was able to use a setup using an intermediate language file in Weblate 4.14:
* File mask: `*.rc`
* Source language: `English`
* Monolingual base language file: `en.rc`
* Edit base file: Yes
* Intermediate language file: `src.rc`

However, when I attempt to set up a component like this in 5.8.3, I get
> Editing template is not supported with this file format

This seems to happen due to the check added in acc61eb5dfd1e6640b92759f6e1a4ec02ea958dc.

Should the check added in acc61eb5dfd1e6640b92759f6e1a4ec02ea958dc apply to my setup with an intermediate language file?
If so: Is there a different way to set up a component using Windows RC files and still be able to edit all languages?

This was the setup that seemed to work in Weblate 4.14:
<img width="303" alt="working_in_v4" src="https://github.com/user-attachments/assets/bd030d99-8bf8-42b6-8925-66dc27f60de5">

**Comment by nijel** [comment]
No editing of source RC files is not supported.

**Reply by JustSwiftIt** [reply]
Thanks for your quick answer. Is there any starting point that you would recommend (in documentation or source code) to help understand why these formats have this restriction and why it also applies when working with an intermediate language file?
